ui/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "module": "es6",
     "experimentalDecorators": true,
     "paths": {
+      "ember-cli-flash/*": ["node_modules/ember-cli-flash/index.d.ts"],
```

Comment:
I have no idea why this is needed. Found it here:
https://github.com/poteto/ember-cli-flash/issues/348#issuecomment-775220232